File: challenges/4-fizz-buzz/python/edurs2602/fizzBuzz.py
valor = int(input(""))

for num in range(1, valor + 1):
    if (num % 3) == 0 and (num % 5) == 0:
        print("FizzBuzz")
    elif (num % 3) == 0:
        print("Fizz")
    elif (num % 5) == 0:
        print("Buzz")
    else:
        print(num)

File: challenges/4-fizz-buzz/java/mateussnts/FizzBuzz.java
import java.util.Scanner;

public class FizzBuzz {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.printf("Digite aqui o valor: ");
        int num = sc.nextInt();

        for (int i = 1; i < num + 1; i++) {
            if (i % 3 == 0 && i % 5 == 0) {
                System.out.println("FizzBuzz");
            
            } else if (i % 3 == 0) {
                System.out.println("Fizz");

            } else if ( i % 5 == 0) {
                System.out.println("Buzz");

            } else {
                System.out.println(i);
            }
        }
    }
}
